core/lib/makeswift/components/site-footer/site-footer.tsx

```diff
@@ -0,0 +1,29 @@
+import { MakeswiftComponent } from '@makeswift/runtime/next';
+import { type ComponentPropsWithoutRef } from 'react';
+
+import { Footer as VibesFooter } from '@/vibes/soul/sections/footer';
+import { getComponentSnapshot } from '~/lib/makeswift/client';
+
+import { PropsContextProvider } from './site-footer.client';
+import { COMPONENT_TYPE } from './site-footer.makeswift';
+
+type Props = ComponentPropsWithoutRef<typeof VibesFooter> & {
+  snapshotId?: string;
+  label?: string;
+};
+
+export const SiteFooter = async ({
+  snapshotId = 'site-footer',
+  label = 'Site Footer',
+  sections: _sections,
```

Comment:
Next time alias this to `streamableSections` since using `_` is reserved for variables that are not used

Reply:
Noted. I'll keep that in mind.